docs/privacy-policy.md

```diff
@@ -1,178 +1,157 @@
+TODO: Add GDPR data protection rights.
 
-_In Short: No._
 
-We don't use cookies for tracking technologies, however we use cookies to ensure authorization between the User and our Site, only when the user is logged in.
+## What log data do we collect?
 
+We automatically store certain information when you use the Site, but this
+information doesn't reveal your identity. It comes from default NGINX server
+logs, which we flush every 24 hours, and includes the following data.
 
-## HOW DO WE HANDLE YOUR SOCIAL LOGINS?
+```
+# IP        Date                         Visited page                  Browser Agent
+1.2.3.4 - - [06/Jun/2021:23:06:13 +0000] "GET / HTTP/1.1" 200 5217 "-" "Mozilla/5.0 ..."
+```
 
-_In Short: If you choose to register or log in to our websites using a social media account, we may have access to certain information about you._
+This information is necessary for maintaining the security and operation of
+our Site, and for our internal analytic and reporting purposes.
 
-Our Site offers you the ability to register and login using your third party social media account details (like your Github, Gitlab, or Codeberg logins). When you choose to do this, we will receive certain profile information about you from your social media provider. The profile Information we receive may vary depending on the social media provider concerned, but will often include your name, email address, follower list, profile picture as well as other information you choose to make public, of which only username and email address will be used on the Site.
 
-We will use the information we receive only for the purposes that are described in this privacy policy or that are otherwise made clear to you on the Site. Please note that we do not control, and are not responsible for, other uses of your personal information by your third party social media provider. We recommend that you review their privacy policy to understand how they store, use and share your personal information, and how you can set your privacy preferences on their Site and apps.
+## Do we use cookies?
 
-## IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
+Yes, we use cookies for the purpose of keeping you signed in and authorizing
+various actions on the Site that require an account.
 
-_In Short: No, not to our knowledge._
 
-Our server is provided by vultr.com, we recommend that you review their privacy policy to understand how they collect, use and share your personal information. Our server is located in Amsterdam, Netherlands. All traffic from userstyles.world is processed by this server.
+## Do we use any external cookies?
 
-If you aren’t a resident in the European Economic Area, then these countries may not have data protection or other laws as comprehensive as those in your country. We will however take all necessary measures to protect your personal information in accordance with this privacy policy and applicable law. 
+No. And we never will.
```

Comment:
"never" you're promising something.

Reply:
Yeah, but are we ever going to use cookies from third parties?

Reply:
Not _likely_, you can't predict the future. And if you now say never will and later on we for some absurd reason needs to it's very conflicting feelings.